__tests__/integration.js

```diff
@@ -82,6 +84,29 @@ test('--mutex network', async () => {
   ]);
 });
 
+test('--mutex network 2', async () => {
```

Comment:
Let's name this `'stress test for "--mutex network"'`?

Alternatively, just use this one remove the old test if it is not enough/useful since this should cover that scenario already.